src/data/javascript-quiz.ts

```diff
@@ -1837,6 +1837,16 @@ const javascriptQuiz = [
     Explanation:
       "JavaScript has 7 primitive data types. Boolean, null, undefined, Number, BigInt, String and Symbol",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_Types#data_structures_and_types"
+  },
+  {
+    Question: "What is the Sum of this Expression (true + 34)",
+    Answer: "35",
+    Distractor1: "34",
+    Distractor2: "true34",
+    Distractor3: "35",
```

Comment:
distractors are for wrong answers only

```suggestion
    Distractor3: "25",
```